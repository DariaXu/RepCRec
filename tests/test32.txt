// Test that a transaction can abort while blocked.
begin(T1)
begin(T2)
R(T2, x2)
// Should be blocked now
W(T1, x2, 8) 
end(T1)
end(T2)
// ---
// assertAborted(T1)
// assertCommitted(T2)
