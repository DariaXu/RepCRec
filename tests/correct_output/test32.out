T2 reads - x2: 20
Transaction T1 blocked by a lock conflict.
Abort: T1
Commit: T2
